### PR TITLE
Samsung.ThemeDesigner version 2.0.4

### PR DESCRIPTION
**Pull request by erikiorai**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/109972&drop=dogfoodAlpha

### CODE REVIEW

**Comment by erikiorai** [comment]
@microsoft-github-policy-service agree

**Comment by erikiorai** [comment]
> @erikiorai please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

**Comment by erikiorai** [comment]
It dosn't download the depedency for same reason

**Comment by stephengillie** [comment]
Hi @erikiorai,

> It dosn't download the depedency for same reason

Yes, dependencies aren't currently supported. We're keeping a list of packages requiring dependencies, to merge once support is ready. 

> It needs .NET Framework 3.5 SP1 or newer

And also `Microsoft.DotNet.Framework.DeveloperPack_4`. Unfortunately, only one dependency would be supported.

**Comment by erikiorai** [comment]
> Hi @erikiorai,
> 
> > It dosn't download the depedency for same reason
> 
> Yes, dependencies aren't currently supported. We're keeping a list of packages requiring dependencies, to merge once support is ready.
> 
> > It needs .NET Framework 3.5 SP1 or newer
> 
> And also `Microsoft.DotNet.Framework.DeveloperPack_4`. Unfortunately, only one dependency would be supported.

`Microsoft.DotNet.Framework.DeveloperPack_4` is .Net Framework 3.5 according to one online guide i found

**Comment by stephengillie** [comment]
> `Microsoft.DotNet.Framework.DeveloperPack_4` is .Net Framework 3.5 according to one online guide i found

Yes, that's correct. I had forgotten because of the slightly different name.

**Comment by Trenly** [comment]
[Policy] Blocking Issue

**Comment by stephengillie** [comment]
> Url validation error
> 
> * manifests\s\Samsung\ThemeDesigner\2.0.4
>   
>   * https://img-developer.samsung.com/contents/cmm/Samsung_Theme_Designer_2.0.4.zip
>     
>     * Http status code: BadGateway

image

File: manifests/s/Samsung/ThemeDesigner/2.0.4/Samsung.ThemeDesigner.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+InstallerSwitches:
+  Silent: /S /v"/qn"
+  SilentWithProgress: /S /v"/qn"
+Installers:
+- Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: Samsung_Theme_Designer_2.0.4.exe
+  InstallerUrl: https://img-developer.samsung.com/contents/cmm/Samsung_Theme_Designer_2.0.4.zip
```
Comment (stephengillie) [inline manifests/s/Samsung/ThemeDesigner/2.0.4/Samsung.ThemeDesigner.installer.yaml:18]: Hi @erikiorai,

This URL appears to be down. Is the package available from another?

**Comment by erikiorai** [comment]
> > Url validation error
> > 
> > * manifests\s\Samsung\ThemeDesigner\2.0.4
> >   
> >   * https://img-developer.samsung.com/contents/cmm/Samsung_Theme_Designer_2.0.4.zip
> >     
> >     * Http status code: BadGateway
> 
> image

Yes it also avelable in web archive https://web.archive.org/web/20160401000000*/https://img-developer.samsung.com/contents/cmm/Samsung_Theme_Designer_2.0.4.zip

File: manifests/s/Samsung/ThemeDesigner/2.0.4/Samsung.ThemeDesigner.installer.yaml (hunk 1 above)
Comment (erikiorai) [reply manifests/s/Samsung/ThemeDesigner/2.0.4/Samsung.ThemeDesigner.installer.yaml:18]: Yes its also avelable in web archive https://web.archive.org/web/20160825083554/https://img-developer.samsung.com/contents/cmm/Samsung_Theme_Designer_2.0.4.zip

Comment (Trenly) [reply manifests/s/Samsung/ThemeDesigner/2.0.4/Samsung.ThemeDesigner.installer.yaml:18]: Unfortunately we can't use archive.org links due to their terms of service

Comment (erikiorai) [reply manifests/s/Samsung/ThemeDesigner/2.0.4/Samsung.ThemeDesigner.installer.yaml:18]: i found another link https://softpedia-secure-download.com/dl/e6e98cb2762ace0731a0c7b0e73a73aa/65b2844a/100199952/software/mobile/Samsung_Theme_Designer_2.0.4.zip

**Comment by erikiorai** [comment]
why did it fail now?

**Comment by erikiorai** [comment]
On my pc it installs successfully withaout any user input

**Comment by erikiorai** [comment]
Hmm did you enable .NET Framework 3.5 feature in the test? mabye that's why

**Comment by erikiorai** [comment]
Can you do the test using the .Net framework 3.5 Feature?

File: manifests/s/Samsung/ThemeDesigner/2.0.4/Samsung.ThemeDesigner.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+Installers:
+- Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: Samsung_Theme_Designer_2.0.4.exe
+  InstallerUrl: https://github.com/AndroidCrazyRomTeamProjects/Samsung-legacy-software-archive/raw/main/Samsung_Theme_Designer_2.0.4.zip
+  InstallerSha256: EA3A99CA24A1BD4433CC960523190BEB5A8CCD2703385073E35F16BB24D11575
+ManifestType: installer
+ManifestVersion: 1.4.0
```
Comment (stephengillie) [inline manifests/s/Samsung/ThemeDesigner/2.0.4/Samsung.ThemeDesigner.installer.yaml:21]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/s/Samsung/ThemeDesigner/2.0.4/Samsung.ThemeDesigner.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Samsung.ThemeDesigner
+PackageVersion: 2.0.4
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.4.0
```
Comment (stephengillie) [inline manifests/s/Samsung/ThemeDesigner/2.0.4/Samsung.ThemeDesigner.yaml:8]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/s/Samsung/ThemeDesigner/2.0.4/Samsung.ThemeDesigner.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+PackageIdentifier: Samsung.ThemeDesigner
+PackageVersion: 2.0.4
+PackageLocale: en-US
+Publisher: Samsung Electronics, Co., Ltd
+PackageName: Samsung Theme Designer
+License: (c) Samsung Electronics, Co., Ltd
+ShortDescription: Theme Designer for Samsung Bada OS and feature phones
+ManifestType: defaultLocale
+ManifestVersion: 1.4.0
```
Comment (stephengillie) [inline manifests/s/Samsung/ThemeDesigner/2.0.4/Samsung.ThemeDesigner.locale.en-US.yaml:12]:
```suggestion
ManifestVersion: 1.6.0
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @erikiorai,

One last change - can the manifest version be increased? Otherwise this PR is ready to merge.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-03-13T16:48:21.8734976Z ##[error] Manifest Error: The multi file manifest has inconsistent field values. [ManifestVersion] Value: 1.4.0 File: Sa***ung.ThemeDesigner.locale.en-US.yaml Manifest Error: The multi file manifest has inconsistent field values. [ManifestVersion] Value: 1.4.0 File: Sa***ung.ThemeDesigner.yaml

(Automated response - build 870.)

**Pull request closed without merging**